guides/source/caching_with_rails.md

```diff
@@ -286,90 +349,174 @@ simply be explicit in a comment, like:
 
-The most efficient way to implement low-level caching is using the `Rails.cache.fetch` method. This method does both reading and writing to the cache. When passed only a single argument, the key is fetched and value from the cache is returned. If a block is passed, that block will be executed in the event of a cache miss. The return value of the block will be written to the cache under the given cache key, and that return value will be returned. In case of cache hit, the cached value will be returned without executing the block.
+By leveraging the speed of modern SSDs, like NVMe, Solid Cache is designed to overcome the limitations of traditional memory-only caches like Redis and Memcached. You get the following benefits with With Solid Cache:
 
-Consider the following example. An application has a `Product` model with an instance method that looks up the product's price on a competing website. The data returned by this method would be perfect for low-level caching:
+- Larger Cache Sizes: The cached data is stored on a disk instead of in memory - this allows you to cache large amounts of data without worrying about the high cost. Memory-only caches like Redis and Memcached are expensive to scale because the come with very little storage capacity.
 
-```ruby
-class Product < ApplicationRecord
-  def competing_price
-    Rails.cache.fetch("#{cache_key_with_version}/competing_price", expires_in: 12.hours) do
-      Competitor::API.find_price(id)
-    end
-  end
-end
+- Cost-Effective: Solid Cache is much more cost-effective than traditional memory-only caches. Memory-only caches are expensive to scale because they require a lot of memory. Solid Cache is much cheaper because it uses disk storage which is much cheaper than memory.
+
+- Simplified Infrastructure: No need for additional dependencies like Redis or Memcached. Additionally, Solid Cache uses a relational database to store cached data, making it simpler to integrate into existing database setups.
+
+While SSDs are slightly slower than RAM, the difference is minimal for most applications. SSDs compensate for this by not needing to be invalidated as frequently, since they can store much more data. As a result, there are fewer cache misses on average, leading to fast response times.
+
+Solid Cache uses a FIFO (First In, First Out) caching strategy, where the first item added to the cache is the first one to be removed when the cache reaches its limit. This approach is simpler but less efficient compared to an LRU (Least Recently Used) cache, which removes the least recently accessed items first, better optimizing for frequently used data. However, Solid Cache compensates for the lower efficiency of FIFO by allowing the cache to live longer, reducing the frequency of invalidations and improving overall performance in many scenarios.
+
+Solid Cache is enabled by default in Rails app from 8.0. However, if you'd prefer not to utilize it, you can skip Solid Cache:
```

Comment:
Nitpick: to make it clear that it's the version of Rails and not version of the solid cache gem?

```suggestion
Solid Cache is enabled by default from Rails version 8.0 and onward. However, if you'd prefer not to utilize it, you can skip Solid Cache:
```